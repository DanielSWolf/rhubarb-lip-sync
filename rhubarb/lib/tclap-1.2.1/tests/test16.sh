#!/bin/sh

# failure
../examples/test3  --stringTest one homer -B -Bh > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test16.out; then
	exit 0
else
	exit 1
fi

