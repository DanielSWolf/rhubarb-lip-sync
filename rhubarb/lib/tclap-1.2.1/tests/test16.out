PARSE ERROR: Argument: (--stringTest)
             Couldn't find delimiter for this argument!

Brief USAGE: 
   ../examples/test3  [-f=<float>] ...  [-i=<int>] ... 
                      --stringTest=<string> [-B] [--] [--version] [-h]
                      <string> <string> <fileNameString> ...

For complete USAGE and HELP type: 
   ../examples/test3 --help

